zhaquirks/sonoff/zbminir2.py

```diff
@@ -0,0 +1,86 @@
+"""Sonoff ZBMINIR2 - Zigbee Switch."""
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+import zigpy.types as t
+from zigpy.zcl import foundation
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class CustomSonoffCluster(CustomCluster):
+    """Custom Sonoff cluster."""
+
+    cluster_id = 0xFC11
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        ExternalTriggerMode = ZCLAttributeDef(
+            name="ExternalTriggerMode",
+            id=0x0016,
+            type=t.uint8_t,
+        )
+
+        DetachRelay = ZCLAttributeDef(
+            name="DetachRelay",
+            id=0x0017,
+            type=t.Bool,
+        )
+
+        TurboMode = ZCLAttributeDef(name="TurboMode", id=0x0012, type=t.int16s)
```

Comment:
Let's remove the `name` from the `ZCLAttributeDef`. The Python attribute name is already enough.
And let's change it to be "snake case" to match other attribute definitions.

Suggestion (can be commited through GitHub UI):
```suggestion
        external_trigger_mode = ZCLAttributeDef(
            id=0x0016,
            type=t.uint8_t,
        )
        detach_relay = ZCLAttributeDef(
            id=0x0017,
            type=t.Bool,
        )
        turbo_mode = ZCLAttributeDef(
            id=0x0012,
            type=t.int16s,
        )
```